commands/Make/Suite.ts

```diff
@@ -0,0 +1,120 @@
+/*
+ * @adonisjs/assembler
+ *
+ * (c) AdonisJS
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+import { BaseCommand, args, flags } from '@adonisjs/core/build/standalone'
+import { files, logger } from '@adonisjs/sink'
+import globParent from 'glob-parent'
```

Comment:
I cannot see `glob-parent` as a dependency inside `package.json` file. Have I missed something?

Reply:
Yep 😋
https://github.com/adonisjs/assembler/blob/develop/package.json#L108

Already on the develop branch and used by the make:test command

Reply:
Ahh okay